### ISSUE TITLE
grouped keywords with multi-level nesting are considered errors

### ISSUE THREAD

**Issue by pgj** [borkdude/clj-kondo#182]
When grouped keywords (namespaced keywords that are not tied to a valid namespace) are used with a nesting more than one level, it is considered an error:

```sh
$ echo ':a/b' | clj-kondo --lint -
linting took 37ms, errors: 0, warnings: 0
$ echo ':a/b/c' | clj-kondo --lint -
<stdin>:0:0: error: can't parse <stdin>, [line 1, col 7] Invalid keyword: a/b/c.
linting took 3ms, errors: 1, warnings: 0
$ clj-kondo --version
clj-kondo v2019.05.12-alpha
```

These can be parsed by the compiler without any problems, so I believe `clj-kondo` should also support them.

**Comment by borkdude** [comment]
Hi @pgj, thanks for reporting this issue. 
I've looked into it and found the following:

- From Clojure documentation:
https://clojure.org/reference/reader
> '/' has special meaning, it can be used once in the middle of a symbol to separate the namespace from the name, e.g. my-namespace/foo. '/' by itself names the division function.

- CLJ-1530: https://dev.clojure.org/jira/browse/CLJ-1530

- ClojureScript:
```
ClojureScript 1.10.520
cljs.user=> :foo/bar/baz
Syntax error reading source at (<cljs repl>:1).
Invalid keyword: :foo/bar/baz.
```

- Cursive: https://github.com/cursive-ide/cursive/issues/1692

This leads me to believe that clj-kondo should not support this. My personal take on this is to use a dot instead of a slash for the secondary grouping: `:foo/bar.baz`.
